packages/frontend/src/components/layout/Layout.tsx

```diff
@@ -1,23 +1,105 @@
 import Head from "next/head";
-import React, { type ReactNode } from "react";
+import React, { useEffect, useState, type ReactNode } from "react";
 import Header from "./Header";
+import Footer from "./Footer";
+import { type UserProfile } from "@auth0/nextjs-auth0/client";
+import ProfileNavigationBlock from "../profile/ProfileNavigationBlock";
+import { MdOutlineHistory } from "react-icons/md";
+import { HiOutlineStar } from "react-icons/hi";
+import ProfileUserInfo from "../profile/ProfileUserInfo";
+import { IoMdInformationCircleOutline } from "react-icons/io";
+import { TbLogout2 } from "react-icons/tb";
+import { useRouter } from "next/router";
+import { RiFeedbackLine } from "react-icons/ri";
+import { IoDocumentTextOutline } from "react-icons/io5";
+// import ActionSheetButtons from "../miniapp/MiniAppActionSheetButtons";
+// import ActionSheetMenu from "../miniapp/MiniAppActionSheetMenu";
 // import Navigation from "./Navigation";
 
 interface LayoutProps {
   children: ReactNode;
 }
 
 const Layout: React.FC<LayoutProps> = ({ children }) => {
+  const [showProfile, setShowProfile] = useState(false);
+  const [user, setUser] = useState<UserProfile>();
+  const router = useRouter();
+
+  useEffect(() => {
```

Comment:
u have already defaulted it to `false` why `useEffect` and set to `false` again?